src/Scripting/CSharpTest/ScriptTests.cs

```diff
@@ -33,6 +33,28 @@ public void TestCreateScript()
         }
 
         [Fact]
+        [WorkItem(17779, "https://github.com/dotnet/roslyn/issues/17779")]
+        public async Task TestScriptWithConstVar()
```

Comment:
>public [](start = 8, length = 6)

It would be better to move the test to ScriptSemanticsTests - no need to involve runtime to validate semantic errors.